applications/nrf5340_audio/src/modules/sd_card.c

```diff
@@ -24,20 +24,30 @@ LOG_MODULE_REGISTER(sd_card, CONFIG_MODULE_SD_CARD_LOG_LEVEL);
 static const char *sd_root_path = "/SD:";
 static FATFS fat_fs;
 static bool sd_init_success;
+K_SEM_DEFINE(m_sem_sd_oper_ongoing, 1, 1);
```

Comment:
Yes. There was some concurrency issue when it was left out. It avoids all the concurrency issues for this module. But I think the issue might be resolved on a higher level. It might be possible if I remember correctly

Reply:
*Might be possible to do it in the sd card playback module

Reply:
But would this stop us open and read multiple SD card file at the same time?

Reply:
Yes